Can't be built anymore, neither on host nor in docker, 'c.GlobalBool undefined'

Hi, I have been using this tool for almost 2 years now, so thanks a lot! Unfortunately I can't built it anymore, neither on host (Ubuntu 18.04 with Go 1.12) nor in docker:

```
$ make docker-build token=xxxx
docker build -t todoist --build-arg TODOIST_API_TOKEN=xxxx
Sending build context to Docker daemon  439.8kB
Step 1/11 : FROM golang:alpine
alpine: Pulling from library/golang
89d9c30c1d48: Pull complete 
8ef94372a977: Pull complete 
1ec62c064901: Pull complete 
a47b1e89d194: Pull complete 
bf1a3d234800: Pull complete 
Digest: sha256:9d2a7c5b6447f525da0a4f18efd2cb05bf7d70228f75d713b7a67345f30157ac
Status: Downloaded newer image for golang:alpine
 ---> 3024b4e742b0
Step 2/11 : RUN apk add --update git bash openssl
 ---> Running in 65026ee9ab6f
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
(1/11) Installing ncurses-terminfo-base (6.1_p20190518-r0)
(2/11) Installing ncurses-terminfo (6.1_p20190518-r0)
(3/11) Installing ncurses-libs (6.1_p20190518-r0)
(4/11) Installing readline (8.0.0-r0)
(5/11) Installing bash (5.0.0-r0)
Executing bash-5.0.0-r0.post-install
(6/11) Installing nghttp2-libs (1.39.2-r0)
(7/11) Installing libcurl (7.66.0-r0)
(8/11) Installing expat (2.2.8-r0)
(9/11) Installing pcre2 (10.33-r0)
(10/11) Installing git (2.22.0-r0)
(11/11) Installing openssl (1.1.1d-r0)
Executing busybox-1.30.1-r2.trigger
OK: 30 MiB in 26 packages
Removing intermediate container 65026ee9ab6f
 ---> ea1c5c618462
Step 3/11 : RUN go get github.com/sachaos/todoist
 ---> Running in 4b1300431c84
# github.com/sachaos/todoist
src/github.com/sachaos/todoist/close.go:14:16: cannot range over c.Args() (type cli.Args)
src/github.com/sachaos/todoist/completed.go:30:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/delete.go:13:16: cannot range over c.Args() (type cli.Args)
src/github.com/sachaos/todoist/format.go:50:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/format.go:53:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/format.go:99:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/labels.go:18:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/list.go:62:6: c.GlobalBool undefined (type *cli.Context has no field or method GlobalBool)
src/github.com/sachaos/todoist/main.go:85:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
src/github.com/sachaos/todoist/main.go:89:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
src/github.com/sachaos/todoist/main.go:89:15: too many errors
The command '/bin/sh -c go get github.com/sachaos/todoist' returned a non-zero code: 2
Makefile:22: recipe for target 'docker-build' failed
make: *** [docker-build] Error 2
```

Any ideas?

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,9 +1,11 @@
 FROM golang:alpine
 
 RUN apk add --update git bash openssl
-RUN go get github.com/sachaos/todoist
+RUN mkdir -p $GOPATH/src/github.com/sachaos/todoist
 
 WORKDIR $GOPATH/src/github.com/sachaos/todoist
+
+RUN git clone https://github.com/sachaos/todoist.git .
 
 RUN go install
 ARG TODOIST_API_TOKEN

